examples/apps/ai-collab/src/types/sharedTreeAppSchema.ts

```diff
@@ -3,99 +3,119 @@
  * Licensed under the MIT License.
  */
 
-import {
-	SchemaFactory,
-	Tree,
-	TreeViewConfiguration,
-	type TreeNode,
-} from "@fluidframework/tree";
+import { Tree, TreeViewConfiguration, type TreeNode } from "@fluidframework/tree";
+import { SchemaFactoryAlpha } from "@fluidframework/tree/alpha";
 import { SharedTree } from "fluid-framework";
 
 // The string passed to the SchemaFactory should be unique
-const sf = new SchemaFactory("ai-collab-sample-application");
+const sf = new SchemaFactoryAlpha("ai-collab-sample-application");
 
 // NOTE that there is currently a bug with the ai-collab library that requires us to rearrange the keys of each type to not have the same first key.
 
-export class SharedTreeTask extends sf.object("Task", {
-	title: sf.required(sf.string, {
+export class SharedTreeTask extends sf.object(
```

Comment:
Note: I've added node-level metadata to these schemas, and fixed some typos. But otherwise, most of these changes are whitespace. If you toggle the "Hide whitespace" option in the GitHub UI, these will be easier to review.